instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/package.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-_instruments = ("pymemcache ~= 1.3",)
```

Comment:
After testing, there was a change in 1.3.5 that added version to the client. Before this the instrumentation fails.

Reply:
So for versions < 1.3.5, the instrumentation did not work properly? According to [this](https://github.com/pinterest/pymemcache/blob/master/ChangeLog.rst#new-in-version-135), a `VERSION` command was added. Was curious as to why without this it would cause the instrumentation to break?

Reply:
Yes, so the instrumentation uses .version() in the tests. This leads to a test failing. While I am fairly confident it would work for versions < 1.3.5, the tests don't and I wanted to keep this PRs scope low given all of the issues I am already currently having.

So locking the version to what actually passes the test felt like a good start.